packages/postgresql/row-to-doc.js

```diff
@@ -46,6 +46,7 @@ function convertValue(pgValue, fieldType) {
   case '@cardstack/core-types::boolean':
   case '@cardstack/core-types::date':
   case '@cardstack/core-types::integer':
+  case '@cardstack/core-types::fixed-decimal':
```

Comment:
I literally spent hours looking for this.

Reply:
Ugh, sorry, I realize this is a megatroll. Maybe we can make it throw? The intention was to not break if a table has some field we don't know how to handle.